File: src/main/java/org/example/kotlinWorkshop/java/lesson03/t1_list/t1_1/Main.java
package org.example.kotlinWorkshop.java.lesson03.t1_list.t1_1;

import java.util.List;

public class Main {

  public static void main(String[] args) {
    var classWithImmutableList = new ClassWithImmutableList(List.of("something"));

    classWithImmutableList.getList().add("potato");

    //unreachable code
    System.out.println(classWithImmutableList.getList());
  }
}

File: src/main/kotlin/org/example/kotlinWorkshop/kotlin/lesson03/t1_list/t1_1/Main.kt
package org.example.kotlinWorkshop.kotlin.lesson03.t1_list.t1_1

fun main() {
    val classWithImmutableList = ClassWithImmutableList(listOf("something"))

    //method does not exist
    //classWithImmutableList.list.add("potato")

    println(classWithImmutableList.list)
}